src/pages/detail-page/subpages/tab-panels/MetadataInformationPanel.tsx

```diff
@@ -70,7 +89,7 @@ const MetadataInformationPanel = () => {
       },
       {
         title: "Full Metadata Link",
-        component: <div>full metadata link</div>,
+        component: <PlainLinkBlock url={metadataLink} />,
```

Comment:
so why dont you call `PlainLinkBlock` to be `MetadataLinkBlock`? unless `plain link block` is something called by STAC, I'd suggest a different name for clarity. "Plain" can be replaced by something else.